### PR TITLE
total amount

### PR DESCRIPTION
**Pull request by parasnegi2002**
Targets #2195 
### Description
added Total amount calculation for the Indian Client Invoice 
### Checklist:
- [x] I have performed a self-review of my own code.

### CODE REVIEW

**Comment by Ayush8923** [comment]
@parasnegi2002 Please add the correct issue target id of your PR. You have added the id of another PR.

**Review by Ayush8923 — COMMENTED** [review]
@parasnegi2002 Please add the correct issue target id of your PR. You have added the id of another PR.

File: Modules/Invoice/Resources/views/subviews/edit/invoice-details.blade.php [hunk 1]
```diff
@@ -58,6 +58,15 @@
                         </span>
                     </div>
                 </div>
+                <div class="form-group" v-if="this.client.type == 'indian'">
+                    <div class="d-flex">
+                        <label for="client_id" class="mr-5">total amount:</label>
+                        <span>
+                            {{ $invoice->display_amount . " + " . $invoice->gst ." ₹". " = " }} <span id="totalAmount">{{$invoice->total_amount }}</span><span> ₹ </span>
+                        </span>
```
Comment (Ayush8923) [inline Modules/Invoice/Resources/views/subviews/edit/invoice-details.blade.php:66]: Can we write it like this?
```suggestion
                        <span>
                            {{ $invoice->display_amount . " + " . $invoice->gst ." ₹". " = " }} <span id="totalAmount">{{$invoice->total_amount }} ₹</span>
                        </span>
```

**Review by Ayush8923 — CHANGES_REQUESTED** [review]
Please check the comment.

**Review by Shivam-Samant — APPROVED** [review]
Looks good

File: Modules/Invoice/Resources/views/subviews/edit/invoice-details.blade.php [hunk 2]
```diff
@@ -206,6 +222,13 @@
         },
 
         changePaidAmountListener() {
+            let totalPaidAmount = this.amountPaid + " + " + $("#tds").val() + " = " + (parseInt(this.amountPaid) + parseFloat($("#tds").val()));
+            document.getElementById("totalPaidAmount").innerText = totalPaidAmount;
+            $(".checkIcon").addClass("d-none");
+            console.log($("#totalAmount").text().trim(), (totalPaidAmount.split(/\=/)[1]).trim(), $("#totalAmount").text().trim() == (totalPaidAmount.split(/\=/)[1]).trim())
```
Comment (Ayush8923) [inline Modules/Invoice/Resources/views/subviews/edit/invoice-details.blade.php:228]: Removed debugging code `console.log`.

**Review by Ayush8923 — CHANGES_REQUESTED** [review]

**Review by Ayush8923 — APPROVED** [review]

**Pull request closed without merging**